CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+
+## [unreleased]
+- Adds APIs for managing tenants
+  - adds `/recipe/multitenancy/tenant` PUT
+  - adds `/recipe/multitenancy/tenant/remove` POST
+  - adds `/recipe/multitenancy/tenant/config` GET
+  - adds `/recipe/multitenancy/tenant/list` GET
+- Adds APIs for creating and managing Thirdparty provider config for tenants
```

Comment:
- api 1 -> input will be connectionuri to use for the new app (only create, no update).
- api 2 -> input will be appId. This will take all the input thats used to create a new tenant, except for the tenantId